Update splitting-a-subfolder-out-into-a-new-repository.md (#25960)
github/docs: content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md

  ghec: '*'
shortTitle: Splitting a subfolder
---

> If you create a new clone of the repository, you won't lose any of your Git history or changes when you split a folder into a separate repository.  
> However, it won't keep the branches and tags of the original repository.

Reviewer: ```suggestion
If you create a new clone of the repository, you won't lose any of your Git history or changes when you split a folder into a separate repository.  However, note that the new repository won't have the branches and tags of the original repository.
```
